extensions/blocks/contact-form/components/jetpack-field.js

```diff
@@ -3,60 +3,117 @@
  */
 import classNames from 'classnames';
 import { __ } from '@wordpress/i18n';
-import { Fragment } from '@wordpress/element';
-import { InspectorControls } from '@wordpress/block-editor';
-import { PanelBody, TextControl } from '@wordpress/components';
+import { Fragment, useEffect } from '@wordpress/element';
+import { InspectorControls, BlockControls } from '@wordpress/block-editor';
+import {
+	PanelBody,
+	TextControl,
+	ToggleControl,
+	Disabled,
+	Toolbar,
+	ToolbarButton,
+	Path,
+} from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import JetpackFieldLabel from './jetpack-field-label';
+import renderMaterialIcon from '../../../shared/render-material-icon';
+
+function JetpackField( props ) {
+	const {
+		isSelected,
+		type,
+		required,
+		label,
+		parentBlock,
+		setAttributes,
+		placeholder,
+		padding,
+		spacing,
+	} = props;
+
+	useEffect( () => {
+		if ( parentBlock && parentBlock.attributes.padding !== padding ) {
+			setAttributes( { padding: parentBlock.attributes.padding } );
+		}
+
+		if ( parentBlock && parentBlock.attributes.spacing !== spacing ) {
+			setAttributes( { spacing: parentBlock.attributes.spacing } );
+		}
+	} );
```

Comment:
I think @Copons' [comments](https://github.com/Automattic/jetpack/pull/15362/files#r413960509) apply here too.

Reply:
Thanks, I've removed this code for now since supporting on the front end blows out the scope of this PR.